jasmine/private/jasmine_test.bzl

```diff
@@ -23,11 +22,7 @@ def _impl(ctx):
     )
     files.extend(ctx.files.data)
 
-    fixed_args = [
```

Comment:
I believe that this is a sane default. given other choices either you provide via args or forced to have a config file. perhaps we should generate a config file with this as default when there is no config file?

Reply:
My gut says we stay less opinionated here and leave it to the user to provide either args or a config. This is the same expectation as using jasmine outside of bazel.

Reply:
I agree. I wanted to bring this as the default for those who are already familiar with `jasmine_node_test`.  this was the default there.
https://github.com/bazelbuild/rules_nodejs/blob/80df1aa4cf39782c12ef8a8959f8c96803c74ab6/packages/jasmine/jasmine_runner.js#L55

Reply:
On the bright side, jasmine will at least fail by default if it finds no specs :)

Reply:
though when you do `npx jasmine` it gives you no tests found by not making the same assumption that we do right now.

Reply:
Ohhh. Interesting. It is `const IS_TEST_FILE = /[^a-zA-Z0-9](spec|test)\.(mjs|cjs|js)$/i;` which covers both `.spec` and `_spec`.

@alexeagle wdyt on this one?

Reply:
Easy answer is to start with rnj equivalence?